.gitignore

```diff
@@ -3,3 +3,4 @@
 /static/daemon/lbrynet*
 /static/locales
 yarn-error.log
+package-lock.json
```

Comment:
We're solely using `yarn` for the project since most of our building dependencies use it. Using `npm` instead can break the build. Thus, `package-lock.json` doesn't need to be ignored.

Reply:
:+1: wasn't sure